mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -863,7 +862,9 @@ _PUBLIC_ enum MAPISTATUS emsabp_table_fetch_attrs(TALLOC_CTX *mem_ctx, struct em
 				lpProps.value.l = 0x0;
 				break;
 			case PidTagDisplayName:
-				lpProps.value.lpszW = NULL;
+				/* This value is temporal to workaround PropertyRow_addprop internals */
+				/* It will be set to NULL after the call to PropertyRow_addprop */
+				lpProps.value.lpszW = "t";
```

Comment:
I disagree, this data is going to be sent back to the client, therefore can have an impact on its behavior.
What does Exchange return in this case? NULL, "", MAPI_E_NOT_FOUND?